### ISSUE TITLE
Create documentation where the MET environment variables are mapped to METplus

### ISSUE THREAD

**Issue by bikegeek** [NCAR/METplus#131]
At the METplus tutorial at NCWCP, some attendees were a bit confused about how the MET environment variables where set.  We need to document how the MET environment variables are set via the entries in the METplus conf files.  We should also make sure that at the very least, we name our METplus variables to be consistent with the MET variables, prepended with the MET tool name:

ie .  POINT_STAT_GRID  maps to GRID in the point_stat MET config file

**Comment by DanielAdriaansen** [comment]
I think we have accomplished the following:

> We should also make sure that at the very least, we name our METplus variables to be consistent with the MET variables, prepended with the MET tool name: ie . POINT_STAT_GRID maps to GRID in the point_stat MET config file

As far as a section detailing how MET environment variables are set, does the section in the documentation:
https://ncar.github.io/METplus/Users_Guide/systemconfiguration.html#user-defined-config

handle this?

My opinion is that this issue can probably be closed.

What do you think @bikegeek @jprestop ?

**Comment by bikegeek** [comment]
It looks like the section of documentation Dan linked to addresses the
original issue brought up by the NCWCP tutorial attendee.  @George McCabe
<mccabe@ucar.edu>, is it OK to close this issue???

---------------
Minna Win
National Center for Atmospheric Research
Developmental Testbed Center
Phone: 303-497-8423
Fax:   303-497-8401

**Comment by georgemccabe** [comment]
The section of documentation regarding user defined configs is what can be used if there is an option in a MET config file that is not yet configurable through METplus. However, many of the common MET config options are handled within the wrapper. The "MET Configuration" section of the single wrapper use cases is where we describe the full list of environment variables that are set and which METplus config they correspond to. See the bottom of this section:

https://ncar.github.io/METplus/Users_Guide/met_tool_wrapper/GridStat/GridStat.html#met-configuration

We have discussed putting this info in another section so it is found in one place, but for now pointing users to the "MET Configuration" section of the met_tool_wrapper use case for a given tool will give users the information they need.

Long story short, I will close this issue.
